### PR TITLE
Assignment4 

### PR DESCRIPTION
**Pull request by marialucas96**

### CODE REVIEW

**Comment by dachafra** [comment]
ok assign4. @ocorcho will review the rest of the assign

File: Assignment2/marialucas96-MariaLucas.turtle.rdf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+	
+@base <http://www.ontogrid.net/StickyNote#>.
+@prefix xmls: <http://www.w3.org/2001/XMLSchema#>.
+
+:Class01 :includes :Sensor029; :includes :Computer101.
+:Sensor029 :hasMeasurement :Measurement8401.
+:Measurement8401 :hasTemperature 29^^xmls:integer; :atTime 2010-06-12T12:00:12^^xmls:dateTime.
```
Comment (ocorcho) [inline Assignment2/marialucas96-MariaLucas.turtle.rdf]: Still lots of issues with the syntax of this Turlte file. I cannot understand why you are still making these mistakes, which already appeared in other student's assignment.

File: Assignment2/marialucas96-Assignment2.ttl [hunk 2]
```diff
@@ -0,0 +1,8 @@
+@base <http://www.ontogrid.net/StickyNote#>.
+@prefix xmls: <http://www.w3.org/2001/XMLSchema#>.
+
+:Class01 :includes :Sensor029; :includes :Computer101.
+:Sensor029 :hasMeasurement :Measurement8401.
+:Measurement8401 :hasTemperature 29^^xmls:integer; :atTime 2010-06-12T12:00:12^^xmls:dateTime.
```
Comment (ocorcho) [inline Assignment2/marialucas96-Assignment2.ttl]: This is still wrong

File: Assignment2/marialucas96-Assignment2.ttl [hunk 3]
```diff
@@ -0,0 +1,8 @@
+@base <http://www.ontogrid.net/StickyNote#>.
+@prefix xmls: <http://www.w3.org/2001/XMLSchema#>.
+
+:Class01 :includes :Sensor029; :includes :Computer101.
+:Sensor029 :hasMeasurement :Measurement8401.
+:Measurement8401 :hasTemperature 29^^xmls:integer; :atTime 2010-06-12T12:00:12^^xmls:dateTime.
+:Computer101 :hasOwner :User10A.
+:User10A: hasName:"Pedro"^^xmls:string.
```
Comment (ocorcho) [inline Assignment2/marialucas96-Assignment2.ttl]: Same here

**Comment by ocorcho** [comment]
There are still errors in the Turtle file. This is not acceptable at this stage, having so many examples of correct TTL files in the repository.

**Pull request closed without merging**